libwild/src/aarch64.rs

```diff
@@ -87,6 +88,11 @@ impl crate::arch::Arch for AArch64 {
     fn local_symbols_in_debug_info() -> bool {
         false
     }
+
+    fn tcb_placement() -> TCBPlacement {
+        // TODO: double-check
+        TCBPlacement::BeforeTP
```

Comment:
Based on my hacky experiment (where I commented out all TLS-related relaxations), a static linking of a trivial main program fails! Will take a look tomorrow.

Reply:
Alright, I've linked a GitHub issue, and I'd like to merge this PR.